concepticondata/conceptlists/Zhang-2019-308.tsv

```diff
@@ -0,0 +1,309 @@
+Zhang-2019-308-106	106	the pig	pig	1337	PIG	
+Zhang-2019-308-107	107	the pillow	pillow	1044	PILLOW	
+Zhang-2019-308-108	108	the plant root	plant root	670	ROOT	
+Zhang-2019-308-109	109	the pliers	pliers			
+Zhang-2019-308-110	110	the poison	poison	655	POISON	
+Zhang-2019-308-111	111	the pulse, beans	pulse, beans	832	BEAN	
+Zhang-2019-308-112	112	the rabbit, hare	rabbit, hare	2345	LEPORID (RABBIT OR HARE)	
+Zhang-2019-308-113	113	be rich	rich	712	RICH	
+Zhang-2019-308-114	114	the rope	rope	1218	ROPE	
+Zhang-2019-308-115	115	the segment	segment	594	PIECE	
+Zhang-2019-308-116	116	seven	seven	1704	SEVEN	
+Zhang-2019-308-117	117	the shame	shame	1776	SHAME	
+Zhang-2019-308-118	118	the sheep	sheep	1331	SHEEP	
+Zhang-2019-308-119	119	the shell	shell	598	SHELL	
+Zhang-2019-308-120	120	the shoulder	shoulder	1482	SHOULDER	
+Zhang-2019-308-121	121	the shoulder blade	shoulder blade	172	SHOULDERBLADE	
+Zhang-2019-308-122	122	be single	single	1493	ONE	"It is actually ""one"" in Sino-Tibetan."
+Zhang-2019-308-123	123	six	six	1703	SIX	
+Zhang-2019-308-124	124	the snake brood	snake brood			
+Zhang-2019-308-125	125	the son-in-law	son-in-law	1056	SON-IN-LAW	
+Zhang-2019-308-126	126	be sour	sour	1906	SOUR	
+Zhang-2019-308-127	127	the sovereign, queen	sovereign, queen	1306	QUEEN	
+Zhang-2019-308-128	128	be stiff	stiff			
+Zhang-2019-308-129	129	to strike	strike	353	STRIKE	
+Zhang-2019-308-130	130	the string, cord	string, cord	362	STRING	
+Zhang-2019-308-131	131	be strong, hard	strong, hard	1884	HARD	
+Zhang-2019-308-132	132	to suffer defeat	suffer defeat	1660	LOSE	
+Zhang-2019-308-133	133	the tail	tail	1220	TAIL	
+Zhang-2019-308-134	134	there is no X such that X...	there is no X such that X...			don't know what to do with this one
+Zhang-2019-308-135	135	the thighbone	thighbone	800	THIGH	
+Zhang-2019-308-136	136	be thin	thin	2249	THIN (OF SHAPE OF OBJECT)	
+Zhang-2019-308-137	137	thou	thou	1215	THOU	
+Zhang-2019-308-138	138	three	three	492	THREE	
+Zhang-2019-308-139	139	to beat	to beat	2133	STRIKE OR BEAT	
+Zhang-2019-308-140	140	to bend	to bend	2224	BEND (SOMETHING)	
+Zhang-2019-308-141	141	to break (vi)	to break (vi)	680	BREAK (DESTROY OR GET DESTROYED)	
+Zhang-2019-308-142	142	to break (vt)	to break (vt)	680	BREAK (DESTROY OR GET DESTROYED)	
+Zhang-2019-308-143	143	to carry water on back	to carry water on back			too specific to map
+Zhang-2019-308-144	144	to chase	to chase	1047	PURSUE	
+Zhang-2019-308-145	145	to chew	to chew	321	CHEW	
+Zhang-2019-308-146	146	to confer	to confer			
+Zhang-2019-308-147	147	to curve	to curve	2224	BEND (SOMETHING)	
+Zhang-2019-308-148	148	to defeat	to defeat	782	DEFEAT	
+Zhang-2019-308-149	149	to descend	to descend	2014	GO DOWN (DESCEND)	
+Zhang-2019-308-150	150	to destroy	to destroy	681	DESTROY	
+Zhang-2019-308-151	151	to die	to die	1494	DIE	
+Zhang-2019-308-152	152	to dig out (earth)	to dig out (earth)	1418	DIG	
+Zhang-2019-308-153	153	to drop, drop	to drop, drop	808	DRIP (FALL IN GLOBULES)	
```

Comment:
This must be specifically expressed through the Chinese lexemes but I can't judge by the English gloss if it should be mapped to DRIP (FALL IN GLOBULES)

Reply:
@laiyunfan

Reply:
After checking the data, I confirm the current mapping.